security-filter/src/main/java/pro/javatar/security/oidc/utils/JsonMessageBuilder.java

```diff
@@ -1,24 +1,39 @@
 package pro.javatar.security.oidc.utils;
 
 import static java.time.format.DateTimeFormatter.ISO_DATE_TIME;
+import static pro.javatar.security.oidc.utils.StringUtils.isBlank;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pro.javatar.security.api.config.SecurityConfig;
 import pro.javatar.security.oidc.exceptions.AuthenticationException;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.stereotype.Component;
 
 import java.time.LocalDateTime;
 import java.util.HashMap;
 import java.util.Map;
 
-@Component
+@Service
 public class JsonMessageBuilder {
+
+    private static final String DEFAULT_ERROR_DESCRIPTION_LINK = "http://jira.javatar.pro/confluence/x/TgZmAQ";
+
     private static final ObjectMapper objectMapper = new ObjectMapper();
-    @Value("${error.description.link: http://jira.javatar.pro/confluence/x/TgZmAQ}")
+
     private String descriptionLink;
 
+    @Autowired
+    public JsonMessageBuilder(SecurityConfig securityConfig) {
+        String descriptionLink = securityConfig.errorDescriptionLink();
+        if (isBlank(descriptionLink)) {
```

Comment:
We need to add method defaultIfBlank(str, DEFAULT) into our util to avoid if-else constructions.

Reply:
done